drivers/gles3/storage/material_storage.cpp

```diff
@@ -1178,7 +1178,7 @@ MaterialStorage::MaterialStorage() {
 		actions.usage_defines["SCREEN_PIXEL_SIZE"] = "@SCREEN_UV";
 		actions.usage_defines["NORMAL"] = "#define NORMAL_USED\n";
 		actions.usage_defines["NORMAL_MAP"] = "#define NORMAL_MAP_USED\n";
-		actions.usage_defines["LIGHT"] = "#define LIGHT_SHADER_CODE_USED\n";
+		actions.usage_defines["LIGHT"] = "#define LIGHT_CODE_USED\n";
```

Comment:
This should be removed. `LIGHT_CODE_USED` is already added by the shader compiler https://github.com/godotengine/godot/blob/0bcc0e92b3f0ac57d4c4650722f347593a258572/drivers/gles3/shader_gles3.cpp#L179

Reply:
Thanks！